src/cli/config.js

```diff
@@ -75,6 +79,17 @@ async function getDefaultConfigurationFileTraversingParents(folder) {
     }
 }
 
+async function getConfigurationInHomedir() {
```

Comment:
Just one last request to add documentation for this function.

```suggestion
/**
 * Get the configuration file to use from the home directory.
 * @returns {Promise<String,Boolean>} The path to the configuration file, or false
 */
async function getConfigurationInHomedir() {
```